### ISSUE TITLE
From facing camera

### ISSUE THREAD

**Issue by Majkwin** [Kaijun/Acer-V5-573G-Hackintosh#43]
Hello
@Kaijun , after installing Yours newest DSDT files I noticed that front facing camera stopped working. I though it was related with updating to Sierra, but now I'm back on El capitan and camera still doesn't work. Could You give me some hints what could be wrong?

**Comment by Kaijun** [comment]
What happens if you open facetime?
On my side, the camera turns on and works if i start facetime.
In addition, camera is also successfully started in browser using HTML5. Demo: https://davidwalsh.name/demo/camera.php 

How about yours?

**Comment by Majkwin** [comment]
With updated DSDT's system doesn't see any conected camera. I know that on old ones it was working.

**Comment by Kaijun** [comment]
![image](https://cloud.githubusercontent.com/assets/1693027/18610678/931694bc-7d22-11e6-97cd-4e09b7c27884.png)

Thats' my camera, obviously it's working.
What happens if you open facebook?

**Comment by Majkwin** [comment]
![bez nazwy](https://cloud.githubusercontent.com/assets/8678532/18610696/17fddf78-7d23-11e6-9c8b-ada0f905f0a7.png)

This is what system report says: "No picture capture device was found" (translated). So I assume system doesn't see any connected camera...

**Comment by Kaijun** [comment]
No idea what's going on with yours.
![image](https://cloud.githubusercontent.com/assets/1693027/18610726/c4aa7a2e-7d23-11e6-9a22-6e932d29043a.png)
Mine works fine.

**Comment by Kaijun** [comment]
maybe SMBIOS?

**Comment by Majkwin** [comment]
No, SMBIOS is fine, just like I said i was using this config all the time :) Facetime and iMessage works. I even tried updating BIOS to 2.30 but it didn't help. Can You tell me Kaijun what did You modify in newest DSDT's? Only graphics ID?

**Comment by Kaijun** [comment]
... after some reboots, my camera also doesn't work anymore.
It seems that this happens accidentally...
![image](https://cloud.githubusercontent.com/assets/1693027/18615505/db5444aa-7da8-11e6-80a0-6eba7045c928.png)

**Comment by macmeister1967** [comment]
I have macOS Sierra 10.12.4, and camera works after creating a single profile with CamTwist...

![download](https://cloud.githubusercontent.com/assets/26684321/24529948/80401d24-15fa-11e7-8e40-1f917d4d0f96.png)

**Comment by macmeister1967** [comment]
PS. I'm using V5-573 (NON-G) model with SuYin webcam...

![untitled](https://cloud.githubusercontent.com/assets/26684321/24530001/d1ea5f90-15fa-11e7-9b4f-2461bc758bfe.jpeg)

**Comment by Kaijun** [comment]
@macmeister1967 Thanks would give a try

**Comment by macmeister1967** [comment]
Strange how it is on the USB3.0 BUS ?!
